tools/shaderc/shaderc_hlsl.cpp

```diff
@@ -563,6 +563,16 @@ namespace bgfx { namespace hlsl
 			return false;
 		}
 
+		char profileAndType[100];
+		memset(profileAndType, 0, 100);
+		if (_options.shaderType == 'f') {
+			profileAndType[0] = 'p';
+		}
+		else {
+			profileAndType[0] = _options.shaderType;
+		}
+		strcat(profileAndType, profile);
```

Comment:
Use `bx::strCat`.

Reply:
@bkaradzic CI failed on MINGW build but the errors seem to be unrelated to my changes (failed on some SIMD instrinsics, etc) any idea on the root cause of failure?